shanoir-ng-preclinical/src/main/java/org/shanoir/ng/preclinical/subjects/dto/PreclinicalSubjectDtoService.java

```diff
@@ -0,0 +1,67 @@
+package org.shanoir.ng.preclinical.subjects.dto;
+
+import org.shanoir.ng.preclinical.subjects.model.AnimalSubject;
+import org.springframework.stereotype.Service;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Service
+public class PreclinicalSubjectDtoService {
+
+    public AnimalSubject getAnimalSubjectFromPreclinicalDto(PreclinicalSubjectDto dto) {
```

Comment:
I think you can use a mapper here to avoid implementing all this (automatically generated)